deploy/backplane/srep/10-srep-admins-project.ClusterRole.yml

```diff
@@ -103,3 +110,132 @@ rules:
   - serverstatusrequests
   verbs:
   - '*'
+# configmaps management
+- apiGroups:
+  - ""
+  resources:
+  - configmaps
+  verbs:
+  - '*'
+# limitrange and resourcequota management
+- apiGroups:
+  - ""
+  resources:
+  - limitranges
+  - resourcequotas
+  verbs:
+  - '*'
+# pvc resizing and deletion
+- apiGroups:
+  - ""
+  resources:
+  - persistentvolumeclaims
+  verbs:
+  - delete
+  - patch
+# serviceaccount creation and deletion
+- apiGroups:
+  - ""
+  resources:
+  - serviceaccounts
+  verbs:
+  - create
+  - delete
+# services management (enable_service_loadbalancers.asciidoc)
+- apiGroups:
+  - ""
+  resources:
+  - services
+  verbs:
+  - '*'
+# app scaling
+- apiGroups:
+  - ""
+  resources:
+  - replicationcontrollers/scale
+  verbs:
+  - patch
+- apiGroups:
+  - apps
+  resources:
+  - deployments/scale
+  - replicasets/scale
+  - statefulsets/scale
+  verbs:
+  - patch
+- apiGroups:
+  - apps.openshift.io
+  resources:
+  - deploymentconfigs/scale
+  verbs:
+  - patch
+# replicaset deletion
+- apiGroups:
+  - ""
+  resources:
+  - replicasets
+  verbs:
+  - delete
+  - deletecollection
+# imagestream management
+- apiGroups:
+  - image.openshift.io
+  resources:
+  - imagestreams
+  - imagestreamtags
+  - imagetags
+  verbs:
+  - '*'
+- apiGroups:
+  - image.openshift.io
+  resources:
+  - imagestreamimports
+  - imagestreammappings
+  verbs:
+  - create
+# machine management (machinetype changes, master replacement)
```

Comment:
Deletion of machines is okay, but anything else should require elevation.

Also, this is a helpful object to put at the cluster level, rather than NS